facetapi.install

```diff
@@ -73,30 +73,36 @@ function facetapi_schema() {
     ),
     'primary key' => array('name'),
   );
-
+  $schema['facetapi_config'] = array(
```

Comment:
I dont understand why youre duplicating the D7 variable table here. We should be using Backdrop CMI instead, no? If you must use database storage then maybe `state_set()/get()`?